README.md

```diff
@@ -1,6 +1,9 @@
 HaikuTwitter
 ============
 
+![HaikuTwitter screenshot](HaikuTwitter.png "HaikuTwitter")
```

Comment:
Change the "GCC4" below to "a more modern compiler than gcc2".
We currently have gcc5, but that may change in the future.